docs/run_nydus_in_kubernetes.md

```diff
@@ -1,3 +1,9 @@
+# Run Dragonfly & Nydus in Kubernetes
+
+We recommend using the Dragonfly P2P data distribution system to further improve the runtime performance of Nydus images.
```

Comment:
Does this section duplicate with L147?

Reply:
@changweige Yes, on purpose. Because this page is redirected from the top link of README.md, it may be difficult for users to find the content of L147.